### ISSUE TITLE
Stopping an already stopped container throws a serializer exception

### ISSUE THREAD

**Issue by mattzuba** [docker-php/docker-php#337]
The Docker API returns a 304 response but there is no message body; however the swagger API spec (and thus the generated code), expects a valid json body with a `message` attribute.  Since it doesn't json_decode successfully, an unexpected exception is thrown.

**Comment by dontub** [comment]
The spec is as it should be, though the implementation isn't: https://github.com/moby/moby/pull/39253
However until this is fixed docker-php-api should use a patched spec file.

**Comment by thaJeztah** [comment]
Yes, I had done some digging; the internal code generates an error message, but it's not sent back through the API because 304's should not have a body; https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.3.5

https://github.com/moby/moby/pull/39485 was just merge to correct the Swagger (and made the same change as https://github.com/moby/moby/pull/39253)
